content/en/docs/tasks/administer-cluster/change-pv-reclaim-policy.md

```diff
@@ -54,6 +54,15 @@ the corresponding `PersistentVolume` is not be deleted. Instead, it is moved to
 
     where `<your-pv-name>` is the name of your chosen PersistentVolume.
 
+    {{< note >}}
+    On Windows, you must _double_ quote any JSONPath template that contains spaces (not single quote as shown above for bash). This in turn means that you must use a single quote or escaped double quote around any literals in the template. For example:
+
+```cmd
+C:\> kubectl patch pv <your-pv-name> -p "{\"spec\":{\"persistentVolumeReclaimPolicy\":\"Retain\"}}"
```

Comment:
As per the style guide, you should [avoid putting the prompt](https://kubernetes.io/docs/contribute/style/style-guide/#don-t-include-the-command-prompt) in sample command-line blocks.
```suggestion
kubectl patch pv <your-pv-name> -p "{\"spec\":{\"persistentVolumeReclaimPolicy\":\"Retain\"}}"
```
If you need to emphasize the context, I'd do that in a comment line above the sample command.

Reply:
Agree, I've changed it according to the style guide and also made another PR against reference/kubectl/jsonpath.md, as I've taken the example with the command prompt from there. Thanks @sftim!